### PR TITLE
[issue #4533]fix NettyEventExecutor.putNettyEvent() to make it thread…

### PR DESCRIPTION
**Pull request by Justinwins**
The change is clear and simple . see https://github.com/apache/rocketmq/issues/4533 for more details.

### CODE REVIEW

**Comment by Justinwins** [comment]
@zhouxinyu could you pls take a look at this issue ?

File: remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingAbstract.java [hunk 1]
```diff
@@ -567,15 +567,12 @@ public void operationComplete(ChannelFuture f) throws Exception {
...
         private final int maxSize = 10000;
+        private final LinkedBlockingQueue<NettyEvent> eventQueue = new LinkedBlockingQueue<NettyEvent>();
 
         public void putNettyEvent(final NettyEvent event) {
-            int currentSize = this.eventQueue.size();
-            if (currentSize <= maxSize) {
-                this.eventQueue.add(event);
-            } else {
-                log.warn("event queue size [{}] over the limit [{}], so drop this event {}", currentSize, maxSize, event.toString());
+            if (!eventQueue.offer(event)) {
```
Comment (lizhanhui) [inline remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingAbstract.java:574]: eventQueue is not bounded... it never fails...and this breaks the current trade-off impl I mentioned in the issue

Comment (lizhanhui) [reply remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingAbstract.java:574]: This comment: https://github.com/apache/rocketmq/issues/4533#issuecomment-1171878588

**Comment by Justinwins** [comment]
Thx .I found there's a terrible mistake there. I meant to use a bounde queue with a maxsize of 1000...

**Pull request closed without merging**